ci-operator/step-registry/azure/provision/bastionhost/azure-provision-bastionhost-ref.yaml

```diff
@@ -22,7 +22,11 @@ ref:
   - name: VNET_NAME
     default: ""
     documentation: |-
-      The vnet name that is already existing.
+      The vnet name that is already existing, or "PRESERVED_FROM_CLUSTER" to use cluster vnet.
+  - name: BASTION_SUBNET_CIDR
```

Comment:
Seem like we did not reference this ENV var in scripts.

Reply:
yes, should be removed.